vault-validation/pom.xml

```diff
@@ -247,6 +247,12 @@
             <artifactId>slf4j-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <!-- see JCRVLT-680 -->
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
+            <scope>test</scope>
```

Comment:
IMHO this is not only a test scope dependency, but really a runtime one (i.e. client using the validation API will need it). Therefore this should have scope "provided". All consumers probably need a fix as well as "provided" is not transitive.

Reply:
The root cause is that vault-validation sets jackrabbit.version to 2.16.0, and that version predates the switch to commons-collections4.